### ISSUE TITLE
[Bug] Tracing does not work with Electron

### ISSUE THREAD

**Issue by remss** [microsoft/playwright#6985]
I'm very excited about the new tracing feature of version 1.12 but I don't get it work with my electron app.

```
const electronApp = await electron.launch({ args: ['main.js'] });
const window = await electronApp.firstWindow();
window.context.tracing // undefined property
```

Is it supported?

**Comment by mxschmitt** [comment]
`context` is a function, does the following work for you?

```js
  electronApp.context().tracing.start({name: "foo", screenshots: true, snapshots: true})
  // ...
  electronApp.context().tracing.stop({
    path: "trace.zip"
  })
```

(We never tested it with Electron)

**Comment by remss** [comment]
Yes the tracing call works thanks! ([documentation ](https://playwright.dev/docs/api/class-tracing)should be updated for electron case?). 

Now when I try to open the trace file with `npx playwright show-trace trace.zip` the Playwright Trace Viewer window shows a blank page. I press Ctrl+Shift+I and here the devtools console logs, hope it helps.

![traceviewer_issue](https://user-images.githubusercontent.com/1586257/121708384-cc578800-cad7-11eb-935e-efdaab84b0ae.PNG)

In the show-trace  --help I see `-b, --browser <browserType>  browser to use, one of cr, chromium, ff, firefox, wk, webkit (default: "chromium")`, electron seems not supported?

**Comment by mxschmitt** [comment]
This seems now fixed with #7101 on the `next` release. `npm install playwright@next` and/or `npx playwright@next show-trace trace.zip`.
